upgrade/upgrade_provider.go

```diff
@@ -44,18 +51,19 @@ func UpgradeProvider(ctx context.Context, repoOrg, repoName string) (err error)
 	var upgradeTarget *UpstreamUpgradeTarget
 	var goMod *GoMod
 
-	ok := step.Run(ctx, step.Combined("Setting Up Environment",
-		step.Env("GOWORK", "off"),
-		step.Env("PULUMI_MISSING_DOCS_ERROR", func() string {
+	err = stepv2.PipelineCtx(ctx, "Set Up Environment", func(ctx context.Context) {
+		env := func(k, v string) { setEnv(ctx, k, v) }
+		env("GOWORK", "off")
+		env("PULUMI_MISSING_DOCS_ERROR", func() string {
 			if GetContext(ctx).AllowMissingDocs {
 				return "false"
 			}
 			return "true"
-		}()),
-		step.Env("PULUMI_CONVERT_EXAMPLES_CACHE_DIR", ""),
-	))
-	if !ok {
-		return ErrHandled
+		}())
+		env("PULUMI_MISSING_MAPPING_ERROR", "true")
```

Comment:
I misled you when we spoke earlier. I believe we should instead set `PULUMI_EXTRA_MAPPING_ERROR` to true. Turns out that was the setting that finally tripped on my machine for the AWS upgrade.
https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pkg/tfgen/generate.go#L1310 is the relevant behavior; the error message checks with what I saw.

If I read [the bridge generate code](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pkg/tfgen/generate.go) and its commit history correctly, it seems like the bridge errors on a missing mapping by default nowadays, so we probably don't even need to set `PULUMI_MISSING_MAPPING_ERROR` at all.